src/main/groovy/util/ConfigHelper.groovy

```diff
@@ -10,16 +10,37 @@
 package util
 
 import grails.util.Holders
+import org.springframework.boot.info.GitProperties
 
 // See http://jira.codehaus.org/browse/GRAILS-6515
 class ConfigHelper {
 
+    /**
+     * Return the version of the application, as specified in build.gradle.
+     */
+    static String getAppVersion() {
+        // build.app.version is readable by war files, app.version by local builds
+        return Holders.grailsApplication.metadata.getProperty(
+            'build.app.version',
+            String,
```

Comment:
Does this need to be String.class? Or does String imply that?

Reply:
Doesn't need to be String.class anymore, welcome to the new groovy I guess